src/main.py

```diff
@@ -237,6 +241,54 @@ async def info(interaction: Interaction) -> None:
     await bc.send_stats(interaction)
 
 
+# Preview command
+@client.slash_command(dm_permission=False)
+async def preview(
+    interaction: Interaction,
+    uploaded_file: Attachment = SlashOption(description="The file to submit."),
+    submit_message: Optional[str] = SlashOption(
+        required=False, description="The submission message text."
+    ),
+) -> None:
+    """Show a preview of a submission's "Now Playing" embed."""
+    user = interaction.user
+
+    if not discord_utils.is_valid_media(uploaded_file.content_type):
+        await interaction.response.send_message(
+            "You didn't send a valid media type. \nTry again.",
+            ephemeral=True,
+        )
+        return
+
+    attachment_filepath = discord_utils.build_filepath_for_attachment(
+        interaction.id, uploaded_file
+    )
+    random_emoji = random.choice(config.emoji_list)
+    await uploaded_file.save(fp=attachment_filepath)
```

Comment:
```suggestion
    await uploaded_file.save(fp=attachment_filepath)
    random_emoji = random.choice(config.emoji_list)
```
nitpick, but i'd swap these lines since random_emoji is related to the line after this block, and the uploaded file save relates to the lines above this block.